### ISSUE TITLE
Issues with using VGA output

### ISSUE THREAD

**Issue by BadAd84** [borti4938/n64rgb#17]
Hi,

I am trying to set up an N64a to use VGA. I have soldered R/G/B, GND and H/V. I get no picture on the display (and this monitor is very forgiving, it can accept composite, component, etc).

I have the jumper set to allow H/V output. I also ensured the firmware is up to date (noticed a recent commit that was a bugfix for HV output).

Is there something I am missing? do you have a test setup with VGA?

Thanks!

**Comment by borti4938** [comment]
Hey,
sorry, I saw this issue plopping up but never reacted. Totally forgot this.
I don't have an extra setup for testing it but can build one on demand. Nevertheless I'm glad that you found the cause for the problem. I didn't know that some monitors need to have 5V connected.
Best Regards, Peter

**Comment by BadAd84** [comment]
It wasn't 5v going to vga that was the issue (although spec seems to imply
some monitors may need it for the edid eeprom). The right side of the n64a
is mostly outputs and as the customer was wiring the n64a to not use the
MultiAV at all, they soldered directly to HD15/VGA connector. So the 5v on
the board was never connected to the console. It was assumed it was an
output, not an input.

Which means no 5v supply on the board and therefore didn't work.

Honestly, in auto pilot I followed the same process until I actually got it
on the bench today and started investigating the cause.

The documentation I am writing in my fork will cover a VGA only
configuration and appropriate wiring.
